test/doom.d/init.el

```diff
@@ -148,7 +149,8 @@
        ;;lua               ; one-based indices? one-based indices
        markdown          ; writing docs for people to ignore
        ;;nim               ; python + lisp at the speed of c
-       nix               ; I hereby declare "nix geht mehr!"
+       (nix              ; I hereby declare "nix geht mehr!"
+        +tree-sitter)
```

Comment:
This one is not strictly necessary, however it makes testing of `tree-sitter` easier (just open a `.nix` file).